README.md

```diff
@@ -6,8 +6,9 @@ This is the Stackdriver metadata agent.
 
 1. Install runtime dependencies:
 
-       $ sudo apt-get install libyajl2 libboost-program-options1.55.0 \
-         libboost-system1.55.0 libboost-thread1.55.0
+       $ sudo apt-get install libyajl2 libboost-filesystem1.55.0 \
```

Comment:
This is for the health check stuff I suppose?

Reply:
Yes, the health check introduced in #85 actually used non-inlined functions from `libboost_filesystem`, so it linked the library.